README.markdown

```diff
@@ -323,7 +323,9 @@ load-nvmrc() {
     local nvmrc_node_version=$(nvm version "$(cat "${nvmrc_path}")")
 
     if [ "$nvmrc_node_version" != "N/A" ] && [ "$nvmrc_node_version" != "$node_version" ]; then
-      nvm use 
+      nvm use
```

Comment:
this will run `nvm install` when `[ "$nvmrc_node_version" = "$node_version" ]`, which isn't desired.

My [comment](https://github.com/creationix/nvm/issues/1272#issuecomment-255858008) meant that this should simply call `nvm install`, and not call `nvm use` at all.